use_cases/timeseries.rst

```diff
@@ -213,7 +214,7 @@ Next, download sample data:
   wget http://examples.citusdata.com/github_archive/github_events-2015-01-01-{0..5}.csv.gz
   gzip -d github_events-2015-01-01-*.gz
 
-And load it:
+And load it (note that this data requires the database to have UTF8 encoding):
```

Comment:
There are other places where we load sample data with \COPY. Can you check if they need a similar warning?

Reply:
Will do.